common/versioning/version.properties

```diff
@@ -1,3 +1,4 @@
-#Thu Aug 02 12:03:16 PDT 2018
-versionName=3.0.5-RC1
-versionCode=1
+#Mon Nov 02 10:02:30 PST 2020
+versionName=3.0.5-RC1
+versionCode=1
+latestPatchVersion=26

```

Comment:
Ah, windows newlines versus unix newlines.

Reply:
Yes.

Reply:
ohhh this is a newline? I was confused about what this was for too long

Reply:
> ohhh this is a newline? I was confused about what this was for too long

I saw that, so I commented to clarify. ;-)